t/cli/test_access_log.sh

```diff
@@ -183,3 +183,44 @@ fi
 make stop
 
 echo "don't log uninitialized access log variable when the HTTP request is malformed"
+
+# TLS upstream
+
+echo "
+apisix:
+    admin_api_mtls:
+        admin_ssl_cert: '../t/certs/apisix_admin_ssl.crt'
+        admin_ssl_cert_key: '../t/certs/apisix_admin_ssl.key'
+    port_admin: 9180
+    https_admin: true
+nginx_config:
+  http:
+    access_log_format: '\"\$upstream_scheme://\$upstream_host\" \$ssl_server_name'
+" > conf/config.yaml
+
+make run
+sleep 2
+
+curl -k -i https://127.0.0.1:9180/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d \
+    '{"uri":"/apisix/admin/routes/1", "upstream":{"nodes":{"localhost:9180":1},"scheme":"https","type":"roundrobin","pass_host":"node"}}'
+
+curl -i http://127.0.0.1:9080/apisix/admin/routes/1
+sleep 4
+tail -n 2 logs/access.log > output.log
+
+# APISIX
+if ! grep '"https://localhost" -' output.log; then
+    echo "failed: find upstream scheme"
```

Comment:
Prompt message, I think it would be better like this.
```
    echo "failed: failed to find the upstream scheme"
```

Reply:
The prompt is just the title of this test, see: https://github.com/apache/apisix/blob/a7e7d60e3b620410c42a0d773baaaf995b4926a3/t/cli/test_validate_config.sh#L31

Ignore the negative/positive word can make life easier.

Reply:
Anyway, I will tweak the message to make it clear.